src/components/pager/pager.stories.js

```diff
@@ -80,7 +80,12 @@ TableComponent.propTypes = {
 function makeStory(name, themeSelector) {
   const component = () => {
     const totalRecords = number('totalRecords', 100);
-    const pageSize = select('pageSize', OptionsHelper.pageSizes, Pager.defaultProps.pageSize);
+    const pageSize = select('pageSize', {
+      one: 1,
```

Comment:
_Note_: It's not possible to use `1` as a value. As far as I can tell there's a bug in `@storybook/addon-knobs` which makes it the default, we want `10` to be the default.